src/main/java/commandLine/CLI.java

```diff
@@ -134,8 +134,8 @@ public static void clearScreen () {
     }
 
     private void firstMenu () {
-        System.out.println("1- SSSP");
-        System.out.println("2- APSP");
+        System.out.println("1- SSSP"); // Single Source Shortest Path
+        System.out.println("2- APSP"); // All Pairs Shortest Paths
```

Comment:
great jop for all, but i think it a good practice if you print it in the menu as the user may get confused.